src/vkvg_context_internal.c

```diff
@@ -1944,3 +1944,46 @@ void _select_font_face (VkvgContext ctx, const char* name){
 	ctx->currentFont = NULL;
 	ctx->currentFontSize = NULL;
 }
+
+VkvgSurface vkvg_get_target (VkvgContext ctx) {
+    if (ctx->status)
+        return NULL;
+    return ctx->pSurf;
+}
+
+void vkvg_push_group (VkvgContext ctx) {
+    if (ctx->status)
+        return;
+
```

Comment:
flush required here

Reply:
On `vkvg_pop_group` too?

Reply:
set a surface as source will flush the context, because descriptor set can not be updated during a renderpass, but if you only pop to a pattern, the context has to be flushed before popping the surf...

Reply:
Can we mark this conversation as resolved or are there any other problems?